### PR TITLE
[PL-48906] Clock synchronization

### PR DESCRIPTION
**Pull request by brian-f-harness**
## Description

* Please describe your changes: Add Clock synchronization topic. This topic was previously on the [community.harness.io site](https://web.archive.org/web/20230118133155/https://community.harness.io/t/clock-synchronization-ntp-delegate-or-on-prem-as-per-iso-soc2/899) but was not added to HDH
* Jira/GitHub Issue numbers (if any): PL-48906
* Preview links/images (Internal contributors only):  [Clock synchronization](https://6616d426040b5c1e7cecad51--harness-developer.netlify.app/docs/platform/settings/clock-sychronization)

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [x] Successful preview build.
- [ ] Code owner review.
- [x] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Comment by Kpmoy** [comment]
@brian-f-harness can we remove lines 9-15 please.

**Review by Kpmoy — APPROVED** [review]

File: docs/platform/settings/clock-sychronization.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- Use your local NTP server
+- Use Use [Google Public NTP](https://developers.google.com/time)
+
+## Configure NTP for a delegate running on Linux
+
+Most Linux distributions use `systemd`, which comes with NTP for clock synchronization. You can verify `systemd` by running `timedatectl`.
+
+### Install the NTP service
+
+1. If NTP is not present on your host system, you can use `yum`, `apt-get`, or `dnf` to install the NTP service as per your OS. You can also use Chrony, which has flexible implementation of NTP.
```
Comment (brian-f-harness) [inline docs/platform/settings/clock-sychronization.md:29]:
```suggestion
1. If NTP is not present on your host system, you can use `yum`, `apt-get`, or `dnf` to install the NTP service as per your OS. You can also use Chrony, which has a flexible implementation of NTP.
```

File: docs/platform/settings/clock-sychronization.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Recommended NTP configuration for Harness Delegate
+
+- Use your local NTP server
+- Use [Google Public NTP](https://developers.google.com/time)
+
+## Recommended NTP configuration for Harness Self-Managed Enterprise Edition
+
+- Use your local NTP server
+- Use Use [Google Public NTP](https://developers.google.com/time)
```
Comment (brian-f-harness) [inline docs/platform/settings/clock-sychronization.md:21]:
```suggestion
- Use [Google Public NTP](https://developers.google.com/time)
```

**Review by Kpmoy — DISMISSED** [review]
LGTM

File: docs/platform/settings/clock-sychronization.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+     sudo yum install ntp      # for CentOS/RHEL
+     sudo dnf install ntp      # for Fedora
+     ```
+
+    NTP is configured using a `ntp.conf` configuration file. The file is generally located in the `/etc/` directory.
+
+2. You can add multiple NTP servers in the `ntp.conf` file in the format below.
+
+    :::info
+    The prefer option should only be specified once.
```
Comment (douglas-j-bothwell) [inline docs/platform/settings/clock-sychronization.md:48]:
```suggestion
    Add the `prefer` option to one `server` command only in the `ntp.conf` file.
```

**Review by douglas-j-bothwell — DISMISSED** [review]
One optional sug otherwise LGTM

**Pull request merged**